NossoMercadoLivre.Domain/Entities/User.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NossoMercadoLivre.Domain.Entities
+{
+    public class User : BaseEntity
+    {
+        public string Login { get; set; }
```

Comment:
*A complicação do nosso é código é proporcional a complicação da nossa feature. Quanto mais simples, melhor.*
+
*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*
Login e Senha são obrigatórios e a data de criação é automática e não pode ser definida por outra parte da aplicação. Sugiro usar um construtor para receber e apagar todos os métodos de acesso. Já que não tem leitura e nem alteração parcial de nada agora.